webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -930,7 +911,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
               key: 'select-none',
               label: 'Clear selected',
               onClick: () => {
-                handleSelectionChange?.('remove-all');
+                handleSelectionChange?.('remove', [0, settings.pageLimit]);
```

Comment:
i think the changed behavior is not intuitive. `clear selected` should clear all selected items instead of the items in a page. 
Especially when sort order is changed, its tedious to clear items in each page.

https://github.com/determined-ai/determined/assets/109113616/a35f4403-15e4-40c7-95dd-72a6084a61b3